src/utils/clients-service.ts

```diff
@@ -0,0 +1,39 @@
+import { gql, useQuery } from '@apollo/client';
+
+const GET_CLIENTS_LIST = gql`
+  query{
+      users(pageInfo: {offset: 0, limit: 20}){
+      nodes{
+          id
+          name
+          email
+      }
+    }
+  }
+`;
+
+export const useClient = (token) => {
```

Comment:
como esse hook tem os estados da lista de usuário, acho que seria 
```suggestion
export const useUserList = (token) => {
```
ou
```suggestion
export const useUsers = (token) => {
```